site/app/templates/polls/ViewPoll.twig

```diff
@@ -58,21 +59,21 @@
                                    {{ not user_admin and option.getId() in user_selections ? "checked" : "" }}
                                    {{ (not user_admin and poll.isOpen()) ? "" : "disabled" }}
                                    onchange="disableNoResponse();"
-                            />
-                            {% include "misc/Markdown.twig" with {
+                                /></td>
+                            <td class="option-text">{% include "misc/Markdown.twig" with {
                                 "content" : option.getResponse(),
                                 "style": "display: inline"
                             } only %}
                             {% if option.isCorrect() and poll.isReleaseAnswer()%}
-                                <i class="fa-solid fa-square-check correct-tag"></i>
-                            {% endif %}
-                            <br/>
-                            <br/>
-                        </div>
+                                <i class="fa-solid fa-square-check fa-lg correct-tag"></i>
+                            {% endif %}</td>
+                        </tr>
                         {% set index = index + 1 %}
                     {% endfor %}
+                    </table>
                 </fieldset>
                 {% if not user_admin and poll.isOpen() %}
+                    <br/>
```

Comment:
In general, relying upon `<br>` for spacing is a bad idea.  It would be better to do this with CSS instead.

Reply:
right, this should be some leftover code and I haven't deleted yet

Reply:
Targeted the submit button this is for and added padding instead.